.travis.yml

```diff
@@ -2,11 +2,12 @@ language: c
 services: docker
 install:
     - wget https://raw.githubusercontent.com/xenserver/xenserver-build-env/master/utils/travis-build-repo.sh
-script: bash travis-build-repo.sh
+script:
+    - cp vhd-tool.opam opam && bash travis-build-repo.sh
 sudo: true
 env:
     global:
         - REPO_PACKAGE_NAME=vhd-tool
-        - REPO_CONFIGURE_CMD=./configure
```

Comment:
```
REPO_CONFIGURE_CMD=true
```
should work.

Reply:
It seems it's passing with empty string, not with true 🤔